api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -36,12 +54,110 @@
                 public org.pmiops.workbench.model.ParticipantCohortStatus apply(ParticipantCohortStatus participant) {
                     return new org.pmiops.workbench.model.ParticipantCohortStatus()
                             .participantId(participant.getParticipantKey().getParticipantId())
+                            .cohortReviewId(participant.getParticipantKey().getCohortReviewId())
                             .status(participant.getStatus());
                 }
             };
 
     /**
-     * Get all participants for the specified cohortId and cdrVersionId.
+     * Converter function from backend representation (used with Hibernate) to
+     * client representation (generated by Swagger).
+     */
+    private static final Function<CohortReview, org.pmiops.workbench.model.CohortReview>
+            TO_CLIENT_COHORTREVIEW =
+            new Function<CohortReview, org.pmiops.workbench.model.CohortReview>() {
+                @Override
+                public org.pmiops.workbench.model.CohortReview apply(CohortReview cohortReview) {
+                    return new org.pmiops.workbench.model.CohortReview()
+                            .cohortReviewId(cohortReview.getCohortReviewId())
+                            .cohortId(cohortReview.getCohortId())
+                            .cdrVersionId(cohortReview.getCdrVersionId())
+                            .creationTime(cohortReview.getCreationTime().toString())
+                            .matchedParticipantCount(cohortReview.getMatchedParticipantCount())
+                            .reviewedCount(cohortReview.getReviewedCount())
+                            .reviewStatus(cohortReview.getReviewStatus());
```

Comment:
Do you want to add reviewSize here too?

Reply:
Seemed logical instead of running a sql statement to get the count of the review sample ever time a request happens

Reply:
I agree, I just don't see the reviewSize field getting set anywhere.

Reply:
Sorry.. I think i must have misread your question. Done